spec/httparty/request_spec.rb

```diff
@@ -226,6 +226,14 @@
         expect(raw_request.get_fields('cookie')).to eql ["custom-cookie=1234567"]
       end
 
+      it 'should capture cookies returned from a 401 response' do
```

Comment:
Could you explain what is different about this test than the one prior `it 'should maintain cookies returned from a 401 response'`?